src/controls/dateTimePicker/DateTimePickerStrings.ts

```diff
@@ -95,4 +95,10 @@ export class DateTimePickerStrings implements IDateTimePickerStrings {
    * Separator between time of day components (hours, minutes, seconds).
    */
   public timeSeparator: string = strings.DateTimePickerTimeSeparator;
+  /**
+   * error message when text is entered in the date picker
+   *  ?? added for a default label until loc strings are completed
+   */
+   public textErrorMessage: string = strings?.DateTimePickerTextErrorMessage ?? "Please enter a valid date";
```

Comment:
@modern-dev-dude can you please use a localised string if DateTimePickerTextErrorMessage is not defined?

Reply:
Actually, when would strings not be defined?

Reply:
@joelfmrodrigues 
Correct me if I am wrong and I will go update the PR, but it appeared to be the localized strings are manually entered for several different languages.  
So I added the English one in the file loc.en on line 223:
DateTimePickerTextErrorMessage:"Please enter a valid date"

I am assuming if the developer uses other locals than English DateTimePickerTextErrorMessage will be undefined. Unless those are handled elsewhere. 

If you have any recommendations for converting English to the other Locals, I'd be happy to insert them all and remove the undefined check

Reply:
If the key is missing in the translation, we should add it, even with the English translation in it.
In the code of the controls we need to assume that the strings are defined.

Reply:
If it's a new localization key, we can run Azure translation services that will create the needed key in all translations files.

Reply:
is this a service I can run myself? I was trying to take the work off you guys haha

Reply:
And for clarity, The next action item for me is to add the key to all the loc files with the English translation?

Reply:
If you have an Azure AD subscription - you can potentially run it yourself :)
Just leave the key for English, and I will run the script after the PR is merged.

Reply:
Got it. Thanks.